lib/iris/plot.py

```diff
@@ -374,7 +367,7 @@ def _draw_2d_from_bounds(draw_method_name, cube, *args, **kwargs):
         string_axes = {}
 
         for coord, axis_name, data_dim in zip([u_coord, v_coord],
-                                              ['plotxaxis', 'yaxis'],
```

Comment:
How weird.  Was this always 'plotxaxis' or did I change it accidentally?  I certainly don't remember doing that.